src/test/java/com/rocana/lucene/codec/v1/TestBlockPostingsFormat3.java

```diff
@@ -56,12 +56,26 @@
 import org.apache.lucene.util.automaton.CompiledAutomaton;
 import org.apache.lucene.util.automaton.RegExp;
 
-/** 
- * Tests partial enumeration (only pulling a subset of the indexed data) 
+/**
+ * Fork of Lucene's {@link com.rocana.lucene.codec.v1.TestBlockPostingsFormat3}.
+ *
+ * Why we forked:
+ *   - To use existing Lucene tests to test our {@link RocanaLucene50PostingsFormat}.
+ *
+ * What changed in the fork?
+ *   - Use {@link RocanaLucene50PostingsFormat} instead of Lucene's original.
+ *   - Removed trailing whitespace.
+ *   - Changed these javadocs.
+ *
+ * See ROCANA-8229
```

Comment:
Same comment here about referring to internal only JIRA tickets in open sourced code.